kedro-datasets/kedro_datasets/netcdf/netcdf_dataset.py

```diff
@@ -0,0 +1,155 @@
+"""NetCDFDataset loads and saves data to a local netcdf (.nc) file."""
+import logging
+from copy import deepcopy
+from pathlib import Path, PurePosixPath
+from typing import Any, Dict
+
+import fsspec
+import xarray as xr
+from kedro.io.core import (
+    AbstractDataset,
+    DataSetError,
+    get_filepath_str,
+    get_protocol_and_path,
+)
+
+log = logging.getLogger(__name__)
```

Comment:
We call this `logger` in pretty much all datasets, so for consistency: 

```suggestion
logger = logging.getLogger(__name__)
```